modules/UI/videolayout/LocalVideo.js

```diff
@@ -67,23 +70,34 @@ LocalVideo.prototype.changeVideo = function (stream) {
     this.videoStream = stream;
 
     let localVideoClick = (event) => {
-        // TODO Checking the classList is a workround to allow events to bubble
+        // TODO Checking the classes is a workround to allow events to bubble
         // into the DisplayName component if it was clicked. React's synthetic
         // events will fire after jQuery handlers execute, so stop propogation
         // at this point will prevent DisplayName from getting click events.
         // This workaround should be removeable once LocalVideo is a React
         // Component because then the components share the same eventing system.
+        const $source = $(event.target || event.srcElement);
         const { classList } = event.target;
-        const clickedOnDisplayName = classList.contains('displayname')
-            || classList.contains('editdisplayname');
+
+        const clickedOnDisplayName
+            = $source.parents('.displayNameContainer').length > 0;
+        const clickedOnPopover
+            = $source.parents('.connection-info').length > 0;
+        const clickedOnPopoverTrigger
+            = $source.parents('.popover-trigger').length > 0
+                || classList.contains('popover-trigger');
+
+        const ignoreClick = clickedOnDisplayName
+            || clickedOnPopoverTrigger
+            || clickedOnPopover;
```

Comment:
Prevent clicks on InlineDialog, which is not a child of the LocalVideo component, from causing focus.